File: src/targets/node/unirest/fixtures/multipart-form-data.js
const unirest = require('unirest');

const req = unirest('POST', 'http://mockbin.com/har');

req.headers({
  'Content-Type': 'multipart/form-data; boundary=---011000010111000001101001'
});

req.multipart([
  {
    body: 'bar'
  }
]);

req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: src/targets/csharp/httpclient/fixtures/multipart-form-data.cs
var client = new HttpClient();
var request = new HttpRequestMessage
{
    Method = HttpMethod.Post,
    RequestUri = new Uri("http://mockbin.com/har"),
    Content = new MultipartFormDataContent
    {
        new StringContent("bar")
        {
            Headers =
            {
                ContentDisposition = new ContentDispositionHeaderValue("form-data")
                {
                    Name = "foo",
                }
            }
        },
    },
};
using (var response = await client.SendAsync(request))
{
    response.EnsureSuccessStatusCode();
    var body = await response.Content.ReadAsStringAsync();
    Console.WriteLine(body);
}
